packages/parser/src/phase-1-6-integration.spec.ts

```
@@ -0,0 +1,158 @@
+import { parse } from "tpeg-core";
+import {
+  grammarAnnotation,
+  ruleDefinition,
+  grammarDefinition,
+  quotedString,
+  singleLineComment,
+  documentationComment
+} from "./index";
+
+// Helper function for easier testing
+const testParse = <T>(parser: any, input: string) => parse(parser)(input);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)